core/proto/core.proto

```diff
@@ -80,11 +80,12 @@ message Order {
         optional int64 id = 1;
         optional int64 registrationID = 2;
         optional int64 expires = 3;
-        optional bytes error = 4;
+        optional ProblemDetails error = 4;
```

Comment:
This is technically a violation of our deployability guidelines, but because this code is only active in staging, I'm willing to throw a few extra errors during deployment.

Reply:
Sounds good.